get_deps.sh

```diff
@@ -291,6 +292,95 @@ EOF
     git clone -b task-bench https://github.com/elliottslaughter/x10.git "$X10_DIR"/x10
 fi
 
+if [[ $USE_HPX -eq 1 ]]; then
+    export HPX_DIR="$TASKBENCH_DEPS_DIR"/hpx
+    cat >>deps/env.sh <<EOF
+export HPX_DIR="\$TASKBENCH_DEPS_DIR"/hpx
+
+EOF
+
+    mkdir -p "$HPX_DIR"
+
+    cat >>"$HPX_DIR"/env.sh <<EOF
+
+export SOURCE_ROOT = "\$HPX_DIR"/src
+export INSTALL_ROOT = "\$HPX_DIR"/install
```

Comment:
Maybe `HPX_SOURCE_ROOT` and `HPX_INSTALL_ROOT`?

Reply:
I have changed them.